primitives/src/locktime/relative.rs

```diff
@@ -89,6 +112,16 @@ impl LockTime {
     ///
     /// This method will **not** round-trip with [`Self::to_sequence`]. See the
     /// docs for [`Self::from_consensus`] for more information.
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use bitcoin_primitives::{Sequence, relative::LockTime};
+    ///
+    /// // Interpret a sequence number from a Bitcoin transaction input as a relative lock time
+    /// let sequence_number = Sequence::from_consensus(7000);
```

Comment:
```suggestion
    /// let sequence_number = Sequence::from_consensus(7_000);
```

devs are getting older... hard to count zeros...

Reply:
These are now more meaningful values.